.github/workflows/build-ios.yml

```diff
@@ -0,0 +1,186 @@
+name: Build IOS
+on:
+  workflow_call:
+    inputs:
+      version_file:
+        type: string
+        required: true
+      macos_version:
+        type: string
+        required: true
+      xcode_version:
+        type: string
+        required: true
+      version:
+        type: string
+        required: true
+      prefix:
+        type: string
+        required: true
+env:
+  GOPRIVATE: github.com/getlantern
+  S3_BUCKET: lantern
+jobs:
+  build-ios:
+    runs-on: ${{ inputs.macos_version }}
+    permissions:
+      contents: "read"
+      id-token: "write"
+    env:
+      version: ${{ inputs.version }}
+      version_file: ${{ inputs.version_file }}
+      prefix: ${{ inputs.prefix }}
+      AC_USERNAME: accounts@getlantern.org
+      AC_PASSWORD: ${{ secrets.AC_PASSWORD }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          lfs: true
+      - name: Pull LFS objects
+        run: git lfs pull
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: "go.mod"
+
+      - name: Setup Xcode
+        uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: ${{ inputs.xcode_version }}
+
+      - name: Repo access
+        run: |
+          mkdir /tmp/cache
+          echo "machine github.com login ${{ secrets.GH_TOKEN }} password x-oauth-basic" > /tmp/cache/.netrc
+          chmod 600 /tmp/cache/.netrc
+
+      - name: Setup Sentry CLI
+        uses: mathieu-bour/setup-sentry-cli@v1
+        with:
+          version: latest
+          token: ${{ SECRETS.SENTRY_TOKEN }} # from GitHub secrets
+          organization: getlantern
+          project: lantern-ios
+
+      - name: Install Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          channel: "stable"
+          flutter-version-file: pubspec.yaml
+
+      - name: Install the Apple certificate and provisioning profile
+        env:
+          BUILD_CERTIFICATE_BASE64: ${{ secrets.IOS_CERTIFICATE_P12_BASE64 }}
+          P12_PASSWORD: ${{ secrets.IOS_CERTIFICATE_P12_PASS }}
+          BUILD_PROVISION_PROFILE_BASE64: ${{ secrets.IOS_PROVISION_PROFILE_BASE6 }}
```

Comment:
Shouldn't this be IOS_PROVISION_PROFILE_BASE64?

Reply:
No, Its IOS_PROVISION_PROFILE_BASE6, I forgot to add 4 on gitHub secrets 😅 , hope that fine for now